src/components/Heading.js

```diff
@@ -11,9 +9,5 @@ const StyledHeading = styled.h1`
 `;
 
 export default function Heading({text}) {
-  return (
-    <>
-      <StyledHeading>{text}</StyledHeading>
-    </>
-  );
+  return <StyledHeading>{text}</StyledHeading>;
```

Comment:
```suggestion
  return <StyledHeading>{children}</StyledHeading>;
```
Now, have a look into my comment for the Heading inside FormPage.js.